tkg/client/cluster.go

```diff
@@ -279,6 +279,14 @@ func (c *TkgClient) waitForClusterCreation(regionalClusterClient clusterclient.C
 			log.Warningf("warning: Cluster is created successfully, but some packages are failing. %v", err)
 		}
 	}
+
+	if config.IsFeatureActivated(config.FeatureFlagPackageBasedLCM) {
+		log.Info("Creating cluster metadata ConfigMaps...")
+		if err := c.CreateOrUpdateClusterMetadata(regionalClusterClient, workloadClusterClient, false, options.Edition, options.ClusterName, options.TargetNamespace); err != nil {
```

Comment:
Per my understanding, metadata configmap is in the management cluster. Why this is invoked in workload cluster creation?

Reply:
It is existing for both management clusters and workload clusters